omegaconf/omegaconf.py

```diff
@@ -785,6 +787,38 @@ def resolve(cfg: Container) -> None:
             )
         omegaconf._impl._resolve(cfg)
 
+    @staticmethod
+    def missing_keys(cfg: Any) -> Set[str]:
+        """
+        Returns a set of missing keys flatten in a dotlist style.
+        :param cfg: An `OmegaConf.Container`,
+                    or a convertible object via `OmegaConf.create` (dict, list, ...).
+        :return: set of strings of the missing keys.
+        :raises ValueError: On input not representing a config.
+        """
+        if not isinstance(cfg, Container):
+            try:
+                cfg = OmegaConf.create(cfg)
+            except ValidationError:
+                raise ValueError(f"Could not create a config out of {cfg}")
```

Comment:
I think that it would be appropriate to use the `_ensure_container` function here.
```suggestion
        cfg = _ensure_container(cfg)
```

Reply:
Yes. once you do, you can also change your tests to take plain dict/list instead of a Container to simplfy.

Reply:
@omry The tests contain inputs of all types: ListConfig, DictConfig, plain list, dict. I think it doesn't hurt having it so, but if you think it's better with only plain lists/dicts I'll change it 😃

Reply:
Let's test everything with plain dict and list, but keep a single test for DictConfig and a single test for ListConfig.

Reply:
👌 I've simplified it adding two tests at the end one with DictConfig and one with ListConfig.